README.docker.md

```diff
@@ -0,0 +1,74 @@
+Zulip
+=====
+
+Zulip is a powerful, open source group chat application. Written in
+Python and using the Django framework, Zulip supports both private
+messaging and group chats via conversation streams.
+
+Zulip also supports fast search, drag-and-drop file uploads, image
+previews, group private messages, audible notifications,
+missed-message emails, desktop apps, and much more.
+
+Further information on the Zulip project and its features can be found
+at https://www.zulip.org.
+
+Using Zulip with docker
+=======================
+
+To install `docker-compose`, see the official Docker docs https://docs.docker.com/compose/install/.
+
+Download the repo or clone it, enter the repo. Use `docker-compose up` to start Zulip in the container.
+On the first start, the Zulip container will copy the configs to the data volume.
+Stop the containers using `docker-compose stop` or `Ctrl+C`. Modify the config files to your needs.
+The config files are located in your data volume/directory, by default it is `/opt/docker/zulip/zulip`.
+After setting all required config options, start Zulip again with `docker-compose up`.
+
+**Please note: If you change the database configuration, change it in the Zulip configuration and in the `docker-compose.yml`.**
+
+If you have modified the config correctly, you should now have a working Zulip running on port 80/http, 443/https.
+
+Configuration
+=============
+
+The Config files are in the data volume in the folder `settings`.
+By default the config files are located at `/opt/docker/zulip/zulip/settings`.
+
+Please refer to the official Zulip documentation on how to configure Zulip.
+
+Creating or Adding your Certificates
+====================================
+
+If you already have Certificates, rename your key to `zulip.key` and your certificate to `zulip.combined-chain.crt`.
+
+You can generate your self-signed SSL certificates by using the following steps:
+
+**STEP 1**: Create the server private key
+
+```bash
+openssl genrsa -out zulip.key 2048
+```
+
+**STEP 2**: Create the certificate signing request (CSR)
+
+```bash
+openssl req -new -key zulip.key -out zulip.csr
+```
+
+**STEP 3**: Sign the certificate using the private key and CSR
+
+```bash
+openssl x509 -req -days 3650 -in zulip.csr -signkey zulip.key -out zulip.crt
+```
+
+Congratulations! You have now generated the self-signed SSL certificate, it is valid for 10 years.
```

Comment:
This section duplicates the SSL part of README.prod.md; can you pull these out into a separate doc (README.ssl.md?) and link to it from both places?